docs/Contributing.md

```diff
@@ -139,7 +139,6 @@ Please do not:
 
 Merging Pull Requests
 ---------------------
-
```

Comment:
Nit: Don't remove the extra space. [Markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) recommends it, and it's consistent with the rest of this document.